fiat-roles/fiat-roles.gradle

```diff
@@ -15,14 +15,19 @@
  */
 
 dependencies {
-  compile project(':fiat-core')
+  compile project(":fiat-core")
 
   spinnaker.group("retrofitDefault")
 
-  compile spinnaker.dependency('bootWeb')
-  compile spinnaker.dependency('groovy')
+  compile spinnaker.dependency("bootActuator")
+  compile spinnaker.dependency("bootWeb")
+  compile spinnaker.dependency("groovy")
```

Comment:
The thing that changed was the quote style - single to double. Groovy was pulled in when the role syncing stuff was copied over from Gate a few PRs ago. I plan on rewriting those bits to Java 8, but that's more of a cleanup task and I wanted to get everything functioning first.